package/contents/ui/lib/AppletVersion.qml

```diff
@@ -1,17 +1,18 @@
-import QtQuick 2.0
-import QtQuick.Controls 1.0
-import QtQuick.Layouts 1.0
-import org.kde.plasma.core 2.0 as PlasmaCore
-import org.kde.plasma.plasmoid 2.0
+import QtQuick 2.15
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.3
+
+import org.kde.plasma.plasma5support as Plasma5Support
+import org.kde.plasma.plasmoid
 
 Item {
 	implicitWidth: label.implicitWidth
 	implicitHeight: label.implicitHeight
 
 	property string version: "?"
-	property string metadataFilepath: plasmoid.file("", "../metadata.desktop")
+	property string metadataFilepath: Plasmoid.file("", "../metadata.desktop")
```

Comment:
Pretty sure this function was removed.

* Plasma5: https://invent.kde.org/plasma/libplasma/-/blob/kf5/src/scriptengines/qml/plasmoid/appletinterface.h#L347
* Plasma6 root `PlasmoidItem` properties:
  * https://invent.kde.org/plasma/libplasma/-/blob/master/src/plasmaquick/appletquickitem.h
  * https://invent.kde.org/plasma/libplasma/-/blob/master/src/plasmaquick/plasmoid/plasmoiditem.h
* Plasma6 attached `Plasmoid.___` properties:
  * https://invent.kde.org/plasma/libplasma/-/blob/master/src/plasma/applet.h

I used `Qt.resolvedUrl()` in TiledMenu, though the path is relative to the current file instead of the `content/` folder.

https://github.com/Zren/plasma-applet-tiledmenu/commit/7a3b1a3d17cd23c856fe6b41e4fd5cf9cd653e68